data/mods/Aftershock/items/comestibles/alienfood.json

```diff
@@ -132,5 +173,406 @@
+    "price": 0,
+    "material": [ "flesh" ],
+    "volume": "250 ml",
+    "vitamins": [ [ "mutant_toxin", 25 ] ],
+    "parasites": 32,
+    "stack_size": 1,
+    "fun": -20
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "alien_tallow",
+    "name": "alien tallow",
+    "weight": "55 g",
+    "color": "white",
+    "spoils_in": "60 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -10,
+    "healthy": -1,
+    "calories": 496,
+    "description": "A smooth white block of cleaned and rendered animal fat.  It will remain edible for a very long time, and can be used as an ingredient in many foods and projects.",
+    "price": 500,
+    "price_postapoc": 800,
+    "//": "*May* have been commercially traded.",
+    "material": [ "flesh" ],
+    "volume": "250 ml",
+    "charges": 2,
+    "stack_size": 4,
+    "vitamins": [ [ "mutant_toxin", 25 ] ],
+    "fun": -18
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "alien_lard",
+    "name": "alien lard",
+    "copy-from": "alien_tallow",
+    "description": "A smooth white block of dry-rendered animal fat.  It will remain edible for a very long time, and can be used as an ingredient in many foods and projects."
+  },
+  {
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "id": "alien_organs",
+    "symbol": "¨",
+    "name": { "str_sp": "unidentifiable alien organs" },
+    "charges": 15,
+    "volume": "2500 ml",
+    "weight": "250 g",
+    "color": "black",
+    "looks_like": "feces_manure",
+    "description": "An unidentifiable pile of organs,  you have no idea how you'd prepare it.",
+    "snippet_category": "alien_organ",
+    "material": [ "flesh" ],
+    "spoils_in": "8 hours",
+    "healthy": -25,
+    "fun": -25,
+    "use_action": [ "POISON" ],
+    "vitamins": [  ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "alien_cracklins",
+    "name": { "str_sp": "alien cracklins" },
+    "weight": "42 g",
+    "color": "brown",
+    "spoils_in": "7 days",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -2,
+    "healthy": -1,
+    "calories": 270,
+    "description": "Bits of alien fat and skin that have been fried until they are crispy and delicious.",
+    "price": 170,
+    "volume": "100 ml",
+    "vitamins": [ [ "mutant_toxin", 25 ] ],
+    "price_postapoc": 500,
+    "material": [ "flesh" ],
+    "charges": 4,
+    "flags": [ "EATEN_HOT" ],
+    "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "frost_human_cracklins",
+    "name": { "str_sp": "frostburned cracklins" },
+    "weight": "42 g",
+    "color": "brown",
+    "spoils_in": "7 days",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -2,
+    "healthy": -1,
+    "calories": 270,
+    "description": "Green scraps of fat and skin, and while they have been fried to a crisp, they are not delicious.",
```

Comment:
Frostburned or freezerburned? And the description is not quite clear about the damage.